pkg/registry/apis/provisioning/export.go

```diff
@@ -63,12 +62,15 @@ func (c *exportConnector) Connect(
 		return nil, err
 	}
 	ns := repo.Config().GetNamespace()
-	logger := c.logger.With("repository", name, "namespace", ns)
+	logger := slogctx.From(ctx).With("logger", "export-connector", "repository", name, "namespace", ns)
 
 	// TODO: We need some way to filter what we export.
 
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		job, err := c.queue.Add(r.Context(), &provisioning.Job{
+		ctx := r.Context()
+		ctx = slogctx.To(ctx, logger)
```

Comment:
In some places, I do this inline, other places I don't. Any preference? :)
I do think I prefer it like this, so as to make the choice of what context to use explicit, but it might be a little much code.